tests/test_server_proto.py

```diff
@@ -2548,6 +2548,7 @@ async def test_server_proto_fetch_limit_01(self):
                 r"""
                     WITH MODULE test
                     SELECT FL_B {
+                        id,
```

Comment:
Can you also add a separate test for `__tid__`?

Reply:
> Can you also add a separate test for `__tid__`?

`__tid__` is magical and is not selectable explicitly.

Reply:
I added a test for `__type__`, which *can* be selected and is also protected.